packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -53,16 +53,22 @@ describe("Garbage Collection Tests", () => {
     // Update this for individual node to update the initial GC state of that node.
     const emptyGCDetails: IGarbageCollectionSummaryDetails = {};
 
+    // The flag for the close call
```

Comment:
This comment is not very useful. You should either remove it or add more detail as to what this flag is used for.